src/compas_ghpython/artists/meshartist.py

```diff
@@ -61,6 +61,12 @@ def mesh(self, mesh):
         self.datastructure = mesh
 
     def draw(self, color=None):
+        """Deprecated. Use ``draw_mesh()``"""
+        # NOTE: This warning should be triggered with warnings.warn(), not be a print statement, but GH completely ignores that
+        print('MeshArtist.draw() is deprecated: please use draw_mesh() instead')
```

Comment:
@gonzalocasas a late comment, had same issue before, this helped solve the problem:
```
import warnings
warnings.simplefilter('always', DeprecationWarning)
warnings.warn('xxxx is deprecated....', DeprecationWarning)
```
https://stackoverflow.com/questions/20960110/how-do-i-get-warnings-warn-to-issue-a-warning-and-not-ignore-the-line

Reply:
I don't know if it's only on my machine, but Grasshopper seemed to completely ignore the warnings triggered via `warnings` module. Does it work for you?

Reply:
Yep worked for me in both Rhino and GH
Many terminals by default filter everything from warning module

Reply:
Then let's change it, `warnings` is definitely the right way to go ;)